x/profiles/simulation/operations_dtag_transfers.go

```diff
@@ -120,7 +120,10 @@ func randomDTagRequestTransferFields(
 	if err != nil {
 		return simtypes.Account{}, types.DTagTransferRequest{}, true
 	}
-	_ = k.StoreProfile(ctx, receiverProfile)
+	err = k.StoreProfile(ctx, receiverProfile)
+	if err != nil {
+		return simtypes.Account{}, types.DTagTransferRequest{}, true
+	}
```

Comment:
I think this should not have been edited. I checked the failing sim test before editing this and the error was: 
```
failed to execute message; message index: 0: the request recipient does not have a profile yet: invalid request [/home/riccardo/Coding/Forbole/Desmos/x/profiles/keeper/msg_server_dtag_transfers.go:27]
        Comment: MsgRequestDTagTransfer
```

This probably means that something about storing or getting a profile using a DTag is not working correctly with casing. We should add more unit tests to see why this is not working.